### PR TITLE
update deffered handling to be better

### PR DESCRIPTION
**Pull request by refriedchicken**
I was running this and taking multiple screenshots on sites and ran into timeouts on captureScreenshot from the call_command.

Moved send_rpc_requests into handle_info as to not block the call.

### CODE REVIEW

**Comment by refriedchicken** [comment]
After looking into this further, my timeout was actually coming from captureScreenshot, which I have fixed by moving to async: pid.

However I still think this pull request is valid for not being blocked on the send_rpc_requests call.

**Review by andrewvy — APPROVED** [review]
Thanks! I could go either way on this. Both are still valid IMO for `handle_call` since it's a deferred `GenServer.reply` later on. The blocking `Websockex.send_frame` will still happen, but now its processed in another message handler loop.

This changes whether or not the blocking `Websockex.send_frame` is handled in the same loop interval or not.

I will probably merge this, but not until tomorrow when I can re-evaluate. :)

RE: Snapshot timeouts

I had ran into them sporadically, but I couldn't pinpoint the exact cause whether it was an actual delay hitting the timeout limit or a blocked process call. Will dive into figuring the details of that out soon.

**Pull request merged**
